kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala

```diff
@@ -59,6 +59,7 @@ abstract class AbstractOperation(opType: OperationType, session: Session)
   def getBackgroundHandle: Future[_] = _backgroundHandle
 
   def statement: String = opType.toString
+  def redactedStatement: String = opType.toString
```

Comment:
statement？

Reply:
do you mean `redactedstatment`? all lowercase?

Reply:
def redactedStatement: String = statement

Reply:
Thanks.

Reply:
`redactedStatement` moves to `SparkOperation`